potpie/context-engine/adapters/inbound/cli/templates/global_agent_bundle/AGENTS.md

```
@@ -0,0 +1,8 @@
+<!-- potpie-start -->
+Potpie is durable project memory: repo/source mappings, decisions, infra,
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)